site/app/templates/misc/MarkdownArea.twig

```diff
@@ -25,7 +25,7 @@
         <button title="Insert a code segment" type="button" onclick="addMarkdownCode(0, $('#{{ markdown_area_id }}'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Code <i class="fas fa-code fa-1x"></i></button>
         <button title="Insert bold text" type="button" onclick="addMarkdownCode(2, $('#{{ markdown_area_id }}'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Bold <i class="fas fa-bold fa-1x"></i></button>
         <button title="Insert italic text" type="button" onclick="addMarkdownCode(3, $('#{{ markdown_area_id }}'))" class="btn btn-default btn-markdown key_to_click"  tabindex="0">Italics <i class="fas fa-italic fa-1x"></i></button>
-        <button title="Preview Markdown" type="button" onclick="{{ onclick }}" class="btn btn-default btn-markdown key_to_click" tabindex="0">Preview <i class="fas fa-eye fa-1x"></i></button>
+        <button title="Preview Markdown" type="button" onclick="{{ onclick }}" class="btn btn-default btn-markdown key_to_click" tabindex="0" data-initialize_preview="{{initialize_preview is defined ? initialize_preview : false}}">Preview <i class="fas fa-eye fa-1x"></i></button>
```

Comment:
I think this can be `initialize_preview == true` , but I also think it's better to do this outside the .twig when you do `include "misc/MarkdownArea.twig"`

Reply:
why is the variable name `data-initialize_preview` have a dash mixed with an underscore?

Reply:
Oops, good catch. I'll fix that